### DISCUSSION TITLE
Enable system tray on second monitor/second taskbar?

### DISCUSSION THREAD

**Discussion by aescolastico** [valinet/ExplorerPatcher#2127] [Ideas]
Is there a way to make the system tray icons also appear on the secondary taskbar?

![image](https://github.com/valinet/ExplorerPatcher/assets/30540257/69bd9ee3-7222-4d9a-9d2f-b26ab80f0a8f)

**Comment by pyrates999** [comment]
It's never appeared on secondary monitor in any version of windows, so probably no.

**Reply by S1332** [reply]
you could pull the main taskbar over on the secondary monitor. i really need that feature back

**Comment by MightyM8** [comment]
I would love to see this feature.

**Comment by Seochokid** [comment]
I also think this feature is absolutely necessary.

**Comment by suheandzl** [comment]
I want this feature so badly.
